x-pack/plugins/uptime/common/rules/uptime_rule_field_map.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const uptimeRuleFieldMap = {
+  // common fields
+  'monitor.id': {
+    type: 'keyword',
+  },
+  'url.full': {
+    type: 'keyword',
+  },
+  reason: {
+    type: 'text',
+  },
+  // monitor status alert fields
+  'error.message': {
+    type: 'text',
+  },
+  'agent.name': {
+    type: 'keyword',
+  },
+  'monitor.name': {
+    type: 'keyword',
+  },
+  'monitor.type': {
+    type: 'keyword',
+  },
+  'observer.geo.name': {
+    type: 'keyword',
+  },
+  // tls alert fields
+  'cert_status.count': {
```

Comment:
can we make this more verbose?
is it a total count? 
plus may be `tls_certificate.count`

Reply:
I just noticed that elsewhere in Uptime we refer to this information indeed as `cert_status`. The verbiage is used all throughout our certs components. Should we keep this to keep consistency. Also happy to change it, let me know.